tests/integration/test_asynchronous_metrics_pk_bytes_fields/test.py

```diff
@@ -0,0 +1,57 @@
+import time
```

Comment:
integration test is good.
I was mentioning the stateless tests, just because they are even more simple than integration.